taipy/gui/gui_actions.py

```diff
@@ -350,23 +350,23 @@ def invoke_long_callback(
         user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status
             function that is invoked at the end of and possibly during the runtime of *user_function*:
 
-            - The first parameter of this function is set to a `State^` instance.
-            - The second parameter of this function is set to a bool or an int, depending on the
+            - The first argument of this function is set to a `State^` instance.
+            - The second argument of this function is set to a bool or an int, depending on the
               conditions under which it is called:
 
-               - If this parameter is set to a bool value, then:
+               - If this argument is set to a bool value, then:
 
                    - If True, this indicates that *user_function* has finished properly.
                         The last argument passed will be the result of the user_function call.
                    - If False, this indicates that *user_function* failed.
 
-               - If this parameter is set to an int value, then this value indicates
+               - If this argument is set to an int value, then this value indicates
```

Comment:
All of these should remain `parameter`

Reply:
fine working on it , will update the pr with latest commit consisting all the requested changes :)